modules/tests/shared/src/test/scala-2.13/io/circe/ArraySeqSuite.scala

```diff
@@ -0,0 +1,44 @@
+package io.circe
+
+import cats.Eq
+import io.circe.tests.CirceSuite
+import io.circe.syntax.EncoderOps
+import cats.instances.list.catsKernelStdEqForList
+import cats.instances.int.catsKernelStdOrderForInt
+import cats.instances.string.catsKernelStdOrderForString
+import io.circe.testing.CodecTests
+
+import scala.collection.immutable.ArraySeq
+
+class ArraySeqSuite extends CirceSuite {
```

Comment:
I'm open to feedback on the location of these tests. Most other decoders don't seem to get a dedicated suite, but these are version-specific and have a little complexity, so I thought it would be justified.